packages/components/src/date-range-filter-picker/index.js

```diff
@@ -6,6 +6,7 @@ import { Component, createRef } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { Dropdown } from '@wordpress/components';
 import PropTypes from 'prop-types';
+import moment from 'moment';
```

Comment:
Why is this imported here?  `moment` is a fairly heavy package so ideally it wouldn't be a dependency for the published `@woocommerce/components` package.  I see that you are passing along a moment instance and thus checking for it with prop-types.  However by virtue of this import, it means that you are also bundling moment with `@woocommerce/components`.  I don't think the additional prop-type check is worth it for this additional bundle weight.

Reply:
You're right - an import of this size isn't worth the type check. I'll rework it.

Reply:
On second thought - `moment` is already part of `@woocommerce/components` via `DatePicker`, `DateFilter`, `DateRange`, and `D3Chart`.

Reply:
Which in my opinion is unfortunate (see #3289) (moment contributes ~240kb to the package weight).  I still think you should avoid the import here so it's easier to address bundle size later, especially when you can simply use something like the following here:

```js
if ( after.isMoment && after.isMoment() ) {
 // stuff
}
```